README.md

```diff
@@ -0,0 +1,34 @@
+<div align="center">
+  <img src="./brand/dragonfly-rainbow.svg" width="300">
+</div>
+
+# Hypothesis
+
+* [Website](https://hypothesis.works/)
+* [Documentation](https://hypothesis.readthedocs.io/en/latest/)
+* [Source code](https://github.com/hypothesisWorks/hypothesis/)
+* [Contributing](https://github.com/HypothesisWorks/hypothesis/blob/master/CONTRIBUTING.rst)
+* [Community](https://hypothesis.readthedocs.io/en/latest/community.html)
+
+Hypothesis is the canonical property-based testing library for Python. A property-based test asserts something for *all* inputs, and lets Hypothesis generate them — including inputs you may not have thought of.
+
+```python
+from hypothesis import given, strategies as st
+
+
+@given(st.lists(st.integers() | st.floats()))
+def test_matches_builtin(lst):
+    assert sorted(lst) == my_sort(lst)
+```
```

Comment:
I need to figure out how to get black to ignore this code block...I don't *want* two lines of space between the import and the code, it's too much.

Reply:
Oh no. It's from shed. Which means no easy way to configure this.